src/components/selectInput.js

```diff
@@ -35,15 +36,20 @@
     const [disabled, setIsDisabled] = useState(initialIsDisabled);
     const mounted = useRef(false);
     const blancoText = useText(blanco);
-    const modelProperty = getProperty(actionProperty.modelProperty) || {};
+    const modelProperty = getProperty(actionProperty.modelProperty || '') || {};
     const [currentValue, setCurrentValue] = useState(useText(prefabValue));
     const labelText = useText(label);
     const defaultValueText = useText(prefabValue);
     const helperTextResolved = useText(helperText);
     const validationMessageText = useText(validationValueMissing);
     const dataComponentAttributeValue = useText(dataComponentAttribute);
 
-    const { referenceModelId, modelId, kind, values = [] } = modelProperty;
+    const {
+      referenceModelId,
+      modelId = modelContext || '',
```

Comment:
Why do we assign modelContext or an empty string to modelId, why not modelId itself?

Reply:
Good point, we change this in the next MR